server/internal/heavy/components.go

```diff
@@ -293,6 +302,25 @@ func newComponents(ctx context.Context, cfg configuration.Configuration, genesis
 		}
 	}
 
+	// Exporter
```

Comment:
might be it should be separate component?

Reply:
It's a part of a heavy node. you can see, that almost all of the heavy's stuff is inited in this function